jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MutableHttpFields.java

```diff
@@ -433,10 +439,15 @@ public String toString()
         return asString();
     }
 
-    private class ListItr implements ListIterator<HttpField>
+    private class Listerator implements ListIterator<HttpField>
```

Comment:
`ListItr` is what the JDK uses. `Listerator` is weird and seems a typo.
How about `FieldsIterator` or similar?

Reply:
Nah! Listerator conveys exactly what it is.